tests/unittests/utils/test_depth_anything.py

```
@@ -6,6 +6,11 @@
 |
 """
 
+import inspect
+import os
+from types import SimpleNamespace
+from typing import Any, Iterator, List, Optional
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)